package.json

```diff
@@ -11,12 +11,12 @@
     "styled-components": "^4.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "source .env.staging && react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "deploy:clean": "rimraf build",
-    "deploy:build": "npm run build",
+    "deploy:build": "source .env && npm run build",
```

Comment:
@mateuszkocz feel free to propose different solution. The other option I could is is ejecting and using webpack plugin for that which I believe is an overhead for this kind of task.

Reply:
I don't know what problem are you trying to solve. 🤔

Reply:
https://github.com/Appnroll/kudosapp-frontend/pull/56/files#diff-3d47003866cf0c519ea52a6c74ea5486R5
https://github.com/Appnroll/kudosapp-frontend/pull/56/files#diff-fdc94bff37358aaa42bc6af26f51eb9fR29
Remove hardcoded API_URL, I followed https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

Reply:
I'm not talking about environmental variables. This topic is about forced sourcing of values.

Reply:
If you are against that way, please tell me the 'better' way you are not against so that we can improve this part, otherwise there is nothing more I can do about it at this moment.

Reply:
@mateuszkocz do you think a better solution would be to import these variables or use a certain script file for it?